apypie/action.py

```diff
@@ -83,7 +83,7 @@ def _validate(self, params, values, path=None):
                         self._validate(param_description.params, value, self._add_to_path(path, param_description.name))
                 if ((param_description.expected_type == 'boolean' and not isinstance(value, bool) and not (isinstance(value, int) and value in [0, 1]))
                         or (param_description.expected_type == 'numeric' and not isinstance(value, int))
-                        or (param_description.expected_type == 'string' and not isinstance(value, basestring))):
+                        or (param_description.expected_type == 'string' and not isinstance(value, (basestring, int)))):
```

Comment:
Is there more, that should be automatically casted to string?
(I miss `long double` a little)

Reply:
:woman_shrugging: for the specific workaround, `int` was sufficient, so I went with that.